### PR TITLE
[JB][SUPP-697] two new audit events for email verification: emailVeri…

### PR DESCRIPTION
**Pull request by barnesjake**
…ficationRequest and emailVerificationResult

### CODE REVIEW

File: project/SbtUpdatesSettings.scala [hunk 1]
```diff
@@ -17,6 +17,7 @@ object SbtUpdatesSettings {
     // version of the Java Runtime only recognizes class file versions up to 52.0
     // ---
     dependencyUpdatesFilter -= moduleFilter("com.vladsch.flexmark", "flexmark-all"),
+    dependencyUpdatesFilter -= moduleFilter("uk.gov.hmrc", "auth-client"),
```
Comment (barnesjake) [inline project/SbtUpdatesSettings.scala:20]: I put this in as I didn't want to bump up to version 6, the latest, maybe we should do soon though? What are your thoughts

File: conf/application.conf [hunk 2]
```diff
@@ -12,6 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Copyright 2023 HM Revenue & Customs
```
Comment (AndyHWChung) [inline conf/application.conf]: there's already a copyright header above this - it's weird it's been put in again

File: project/SbtUpdatesSettings.scala (hunk 1 above)
Comment (AndyHWChung) [inline project/SbtUpdatesSettings.scala:20]: what's the reason behind this ignore?

File: test/testsupport/ItSpec.scala [hunk 3]
```diff
@@ -181,8 +184,11 @@ trait ItSpec
     barsRepo.collection.deleteMany(BsonDocument("{}")).toFuture().futureValue
   }
 
-  def stubCommonActions(): StubMapping =
+  def stubCommonActions(): StubMapping = {
+    //todo jake check if this is needed, it isn't in frontend?
```
Comment (AndyHWChung) [inline test/testsupport/ItSpec.scala]: did you test this out again?

File: conf/application.conf (hunk 2 above)
Comment (barnesjake) [reply conf/application.conf]: Removed

File: project/SbtUpdatesSettings.scala (hunk 1 above)
Comment (barnesjake) [reply project/SbtUpdatesSettings.scala:20]: There's a new upgrade but a comment above scared me lol. I don't know if the bootstrap version we are on has the same auth-client version as latest available (It only came out a couple of days ago).

File: test/testsupport/ItSpec.scala (hunk 3 above)
Comment (barnesjake) [reply test/testsupport/ItSpec.scala]: Yeah, I've left it in to stop messages in logs when tests run, to keep logs a bit cleaner

File: project/SbtUpdatesSettings.scala (hunk 1 above)
Comment (AndyHWChung) [reply project/SbtUpdatesSettings.scala:20]: I think it looks safe - version 6 removes support for GDS Verify which we're not using anyway

Comment (barnesjake) [reply project/SbtUpdatesSettings.scala:20]: https://github.com/hmrc/bootstrap-play/blob/main/project/LibDependencies.scala#L25
The comment above says we need it to be the same as what bootstrap has, bootstrap is on 5.12 I think

Comment (barnesjake) [reply project/SbtUpdatesSettings.scala:20]: True, I'll remove and bump it up

**Review by AndyHWChung — APPROVED** [review]

**Pull request merged**